.golangci.yml

```diff
@@ -220,13 +222,16 @@ linters:
   - importas                  # Enforces consistent import aliases.
   - ineffassign               # Detects when assignments to existing variables are not used
   - loggercheck               # Checks key value pairs for common logger libraries (kitlog,klog,logr,zap).
+  - makezero                  # Finds slice declarations with non-zero initial length.
   - mirror                    # Reports wrong mirror patterns of bytes/strings usage.
   - misspell                  # Finds commonly misspelled English words in comments.
   - nakedret                  # Checks that functions with naked returns are not longer than a maximum size (can be zero).
   - nilerr                    # Finds the code that returns nil even if it checks that the error is not nil.
+  - nilnil                    # Checks that there is no simultaneous return of nil error and an invalid value.
```

Comment:
IMO we can get rid of this linter as it is no-linted almost everywhere

Reply:
But if you think it is not worth removing feel free to leave it and merge

Reply:
@waodim I would recommend you to check these examples https://github.com/Antonboom/nilnil?tab=readme-ov-file#examples

Reply:
very good point thanks :slightly_smiling_face: 

I am fine with enabling it and the rest as well :+1: